elastic-stack/elasticsearch/configure.sls

```diff
@@ -80,6 +80,15 @@ set_swapiness_for_elasticsearch_node:
 
 # Up the count for file descriptors for Lucene https://www.elastic.co/guide/en/elasticsearch/reference/current/file-descriptors.html
 increase_elasticsearch_file_descriptor_limit:
+  cmd.run:
+    - name: sysctl -w fs.file-max={{ elasticsearch.fd_limit }}
+  file.replace:
+    - name: /etc/sysctl.conf
+    - pattern: '^fs.file-max=.*'
+    - repl: fs.file-max={{ elasticsearch.fd_limit }}
+    - append_if_not_found: True
+    - onchanges_in:
+        - service: elasticsearch_service
```

Comment:
We'll need to split this into different IDs because there are multiple functions in the `file` module which doesn't work in Salt. Probably put the sysctl stuff into an ID that's something like 'update_system_file_descriptor_limits`

Reply:
OK. I've added commit 2037fba to break this into two states.